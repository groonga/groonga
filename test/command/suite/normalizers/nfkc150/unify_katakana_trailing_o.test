normalize \
  'NormalizerNFKC150("unify_katakana_trailing_o", true, \
                     "report_source_offset", true)' \
  "ォオオオコオソオトオノオホオモオョオヨオロオゴオゾオドオボオポオ" \
  WITH_CHECKS|WITH_TYPES
