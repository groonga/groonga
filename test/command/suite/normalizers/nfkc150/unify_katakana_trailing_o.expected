normalize   'NormalizerNFKC150("unify_katakana_trailing_o", true,                      "report_source_offset", true)'   "ォオオオコオソオトオノオホオモオョオヨオロオゴオゾオドオボオポオ"   WITH_CHECKS|WITH_TYPES
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "ォウオウコウソウトウノウホウモウョウヨウロウゴウゾウドウボウポウ",
    "types": [
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana"
    ],
    "checks": [
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0
    ],
    "offsets": [
      0,
      3,
      6,
      9,
      12,
      15,
      18,
      21,
      24,
      27,
      30,
      33,
      36,
      39,
      42,
      45,
      48,
      51,
      54,
      57,
      60,
      63,
      66,
      69,
      72,
      75,
      78,
      81,
      84,
      87,
      90,
      93
    ]
  }
]
